src/main/java/org/snomed/snowstorm/core/data/services/ConceptUpdateHelper.java

```diff
@@ -81,7 +81,9 @@
 
 	private final Logger logger = LoggerFactory.getLogger(getClass());
 
-	PersistedComponents saveNewOrUpdatedConcepts(Collection<Concept> concepts, Commit commit, Map<String, Concept> existingConceptsMap) throws ServiceException {
+	PersistedComponents saveNewOrUpdatedConcepts(Collection<Concept> concepts, Commit commit, 
+			Map<String, Concept> existingTargetConceptsMap,
```

Comment:
Could we name this back to `existingConceptsMap` because that's more intuitive and they are the existing concepts on this branch.
The other map could be called `rebaseExistingSourceConceptsMap` too to make it more obvious?

Reply:
Agree.   I've tried to keep that same existingRebaseSourceComponent naming througout.